core/src/test/java/com/radixdlt/api/core/LtsAccountDepositBehaviourTest.java

```diff
@@ -0,0 +1,154 @@
+                  new LtsStateAccountDepositBehaviourRequest()
+                      .network(networkLogicalName)
+                      .accountAddress(addressing.encode(accountAddress))
+                      .resourceAddresses(
+                          List.of(
+                              addressing.encode(ScryptoConstants.XRD_RESOURCE_ADDRESS),
+                              addressing.encode(
+                                  ScryptoConstants.VALIDATOR_OWNER_TOKEN_RESOURCE_ADDRESS))));
+
+      // Assert:
+      assertThat(result.getBadgeOfAuthorizedDepositor()).isNull(); // was not requested
+      assertThat(result.getDefaultDepositRule()).isEqualTo(DefaultDepositRule.ACCEPT);
+      assertThat(result.getResourceSpecificBehaviours())
+          .isEqualTo(
+              Map.of(
+                  addressing.encode(ScryptoConstants.XRD_RESOURCE_ADDRESS),
+                  new ResourceSpecificDepositBehaviour()
+                      .resourcePreference(null)
+                      .vaultExists(false)
+                      .isXrd(true)
+                      .depositAllowed(true),
+                  addressing.encode(ScryptoConstants.VALIDATOR_OWNER_TOKEN_RESOURCE_ADDRESS),
+                  new ResourceSpecificDepositBehaviour()
+                      .resourcePreference(null)
+                      .vaultExists(false)
+                      .isXrd(false)
+                      .depositAllowed(true)));
+
+      // Follow-up: deposit some actual XRD into that account
+      submitAndWaitForSuccess(test, Manifest.depositFromFaucet(accountAddress), List.of());
+
+      // Act: this time also pass some dummy badge to be checked
+      final var differentResult =
+          getLtsApi()
+              .ltsStateAccountDepositBehaviourPost(
+                  new LtsStateAccountDepositBehaviourRequest()
+                      .network(networkLogicalName)
+                      .accountAddress(addressing.encode(accountAddress))
+                      .badge(
+                          new ResourcePresentedBadge()
+                              .resourceAddress(
+                                  addressing.encode(ScryptoConstants.XRD_RESOURCE_ADDRESS)))
+                      .resourceAddresses(
+                          List.of(addressing.encode(ScryptoConstants.XRD_RESOURCE_ADDRESS))));
+
+      // Assert: a slight change in the response to the same request (XRD vault now exists)
+      assertThat(differentResult.getBadgeOfAuthorizedDepositor())
+          .isFalse(); // badge was given, but no AD list exists
+      assertThat(differentResult.getResourceSpecificBehaviours())
+          .isEqualTo(
+              Map.of(
+                  addressing.encode(ScryptoConstants.XRD_RESOURCE_ADDRESS),
+                  new ResourceSpecificDepositBehaviour()
+                      .resourcePreference(null)
+                      .vaultExists(true)
+                      .isXrd(true)
+                      .depositAllowed(true)));
+    }
```

Comment:
I'm slightly worried we haven't tested that they keys and payloads are serialized correctly from the stores.

I suggest the following two test cases:
* Changes default behaviour to DenyAll & a test which checks if we can submit XRD - this is most important test for the exchange use case.
* Sets default behaviour to Existing, set an authorized depositor badge and a resource preference on the account, and check that the response is correct 

We'll need to add some manifests to `Manifest.java` for these.

And then can submit those manifests with eg:
```java
      var accountKeyPair = ECKeyPair.generateNew();
      var accountAddress = Address.virtualAccountAddress(accountKeyPair.getPublicKey());
      submitAndWaitForSuccess(test, Manifest.zYZ(accountAddress), List.of(accountKeyPair));
```

Reply:
Added 👍 
(almost exactly the 2 scenarios you describe; anyway I am sure that they verify that "the right resource preference is read from the KV" and "the right depositor badge is read from the KV")